vital/types/metadata_traits.cxx

```diff
@@ -94,7 +111,37 @@ ::find( vital_metadata_tag tag ) const
   return *ix->second;
 }
 
-// ------------------------------------------------------------------
+// ----------------------------------------------------------------------------
+vital_meta_trait_base const&
+metadata_traits
+::find_name( std::string name ) const
+{
+  auto ix = m_name_trait_table.find( name );
+  if ( ix == m_name_trait_table.end() )
+  {
+    LOG_INFO( m_logger, "Could not find trait for name: " << name );
+    auto const defualt_ix = m_trait_table.find(VITAL_META_UNKNOWN);
+    return *defualt_ix->second;
```

Comment:
If you're going to "trust" that this item exists (which should be fine), you might as well write:
```suggestion
    return *m_trait_table[ VITAL_META_UNKNOWN ];
```
(Likewise below.)

...or follow the existing pattern. Or make the existing one follow this pattern (and, ahem: `auto const ix = ...`).